### PR TITLE
make 1.6.x build steps like master

### PR DESCRIPTION
**Pull request by Narretz**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?**

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,21 +37,63 @@ addons:
...
+    on_start: always # default: false
+
+jobs:
+  include:
+    - stage: deploy
+      env:
+        - JOB=deploy
+      before_script: skip
+      script:
+        - "./scripts/travis/build.sh"
```
Comment (gkalpak) [inline .travis.yml]: Not directly related to this PR, but I noticed that the build script is run even on PRs (which will never get deployed).
Could we skip the extra work on PRs?

**Review by gkalpak — APPROVED** [review]
LGTM
<sub>(My one comment is not really related to this PR.)</sub>

File: .travis.yml (hunk 1 above)
Comment (Narretz) [inline .travis.yml]: Maybe we can skip the deploy stage on PRs via some config option
Or we can use the TRAVIS_PULL_REQUEST env variable to skip the build script. 
I've added it as a task here https://github.com/angular/angular.js/issues/16106

Comment (gkalpak) [inline .travis.yml]: (Quite surprisingly, I didn't see a config option to skip that for PRs - or maybe I missed that. But yeah, `TRAVIS_PULL_REQUEST` would do the job.)

**Pull request merged**